.github/workflows/_linux_ut.yml

```diff
@@ -234,6 +234,19 @@ jobs:
             test_cmd="${test_cmd} test_xpu.py"
           fi
           eval $test_cmd 2>${{ github.workspace }}/ut_log/torch_xpu/torch_xpu_test_error.log | tee ${{ github.workspace }}/ut_log/torch_xpu/torch_xpu_test.log
+      - name: Run Torch XPU Distributed UT
+        if: contains(inputs.ut, 'torch_xpu_distributed') || github.event_name == 'schedule'
```

Comment:
```suggestion
        if: contains(inputs.ut, 'xpu_distributed')
```

Reply:
Move the schedule into nightly / ondemand workflow to trigger distribute UT job. it can be added in another PR